Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Corefinement/face_graph_utils.h

```diff
@@ -1234,6 +1265,9 @@ void append_patches_to_triangle_mesh(
     }
     set_next(h_out, candidate, output);
   }
+
+  user_visitor.build_output_step();
+
```

Comment:
```suggestion
```

Reply:
What is different?

Reply:
There is nothing interesting here to track.